yaki_mobile/lib/presentation/ui/shared/views/Header.dart

```diff
@@ -0,0 +1,107 @@
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutter/material.dart';
+
+class Header extends StatelessWidget {
+  final String pictoIcon;
+  final String pictoPath;
+  final String headerTitle;
+  final String headerHint;
+
+  //Header({Key? key}) : super(key: key);
+  Header(
+      {super.key,
+      required this.pictoIcon,
+      required this.pictoPath,
+      required this.headerTitle,
+      required this.headerHint});
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+          borderRadius: const BorderRadius.only(
+              bottomLeft: Radius.circular(15),
+              bottomRight: Radius.circular(15)),
+          color: Colors.amber[400],
```

Comment:
use the Figma color code and use also a dedicated color file containing color code constants